Apps/Admin/Client/Utils/StoreUtility.cs

```diff
@@ -28,10 +28,20 @@ public static class StoreUtility
         /// <summary>
         /// Formats errors returned from external requests.
         /// </summary>
-        /// <param name="apiException">An exception returned by Refit.</param>
         /// <param name="resultError">An error returned from the server.</param>
         /// <returns>The generated <see cref="RequestError"/>.</returns>
-        public static RequestError FormatRequestError(ApiException? apiException, RequestResultError? resultError)
+        public static RequestError FormatRequestError(RequestResultError? resultError)
+        {
+            return FormatRequestError(null, resultError);
+        }
+
+    /// <summary>
+        /// Formats errors returned from external requests.
+        /// </summary>
+        /// <param name="exception">An exception returned by Refit.</param>
+        /// <param name="resultError">An error returned from the server.</param>
+        /// <returns>The generated <see cref="RequestError"/>.</returns>
+        public static RequestError FormatRequestError(Exception? exception, RequestResultError? resultError = null)
```

Comment:
Why not default exception to null as well?

Reply:
That would not work as I had to overload with public static RequestError FormatRequestError(RequestResultError? resultError).  Having exception default to null and result error default to null will not allow StoreUtility.FormatRequestError(response?.ResultError) to compile.  Even though exception defaults to null, it is still expecting an exception to be populated.

Here is what the code supports:

- StoreUtility.FormatRequestError(e) - this is with an exception
- StoreUtility.FormatRequestError(response?.ResultError) - this is with ResultError
- StoreUtility.FormatRequestError(response.Error, response.Content?.ResultError) - this continues to support existing implementations.

Reply:
What do you think about exposing 3 public methods that have non-null parameters (one for Exception, one for RequestResultError, one for both), and have all 3 call the private method that accepts both as nullable?

Reply:
Oh, maybe we need to support the nullable on RequestResultError to make it easier to pass in ResultError.